### PR TITLE
Win10

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: templates/windows-10/default-browser-large-cta.html [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+    border-radius: 15px;
+    visibility: hidden;
+  }
+
+  #xbutton-{{ snippet_id }}:hover + #block-snippet-overlay-{{ snippet_id }} {
+    visibility: visible;
+  }
+</style>
+
+<div class="snippet" id="fx-default-{{ snippet_id }}">
```
Comment (glogiotatidis) [inline templates/windows-10/default-browser-large-cta.html]: @ckprice I changed `id`s to `something-{{ snippet_id }}`. Having the id -which is a number- first breaks css

File: templates/windows-10/default-browser-large-cta.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+  <div class="block-snippet-button" id="xbutton-{{ snippet_id }}">
```
Comment (glogiotatidis) [inline templates/windows-10/default-browser-large-cta.html]: fyi `block-snippet-button` classes are handled by snippet service js https://github.com/mozilla/snippets-service/commit/bb3a3c3e5893ba2f25b3cb74b72d6d505ec4ea8d

**Comment by glogiotatidis** [comment]
This adds snippet block css classes and proper indentation. all of the code was reviewed in #49

**Comment by ckprice** [comment]
@glogiotatidis thanks Giorgos!

Wondering if you could check these two CMR's and move to production as well: [Win10](https://snippets.allizom.org/admin/base/clientmatchrule/37/) and [40.0.2+](https://snippets.allizom.org/admin/base/clientmatchrule/36/). These will be both be needed for deployments. And should be added to production.

**Comment by glogiotatidis** [comment]
win10 cmr is already in prod

**Comment by glogiotatidis** [comment]
as well as the 40.0.2 cmr

**Pull request merged**